templates/Detailed_stocks_news/config.js

```diff
@@ -0,0 +1,30 @@
+config = {
+  name:"Detailed Stocks News",
+  description:"Get detailed stocks news based on news ID",
+  author:"Kevin L",
+  configurationSections: [
+    {
+      title: 'Detailed Stocks News',
+      items: [
+        {
+          label: 'Output Variable',
+          type: 'text',
+          variable: 'outputVar',
+          helpText: 'Save the generated image URL to this variable',
```

Comment:
This should say something like "Save the output result text to this variable", there's no image